### ISSUE TITLE
How to track running pnl across positions including brokerage charges?

### ISSUE THREAD

**Issue by abhisheksoni27** [dhan-oss/DhanHQ-py#44]
I currently do not see any such API, or set of APIs that could get us this amount. This can be really helpful in terms of automatically switching off the trading if the ( MTM Profit / Loss ) - (Brokerage) is above or below already set thresholds.

Would love to work on the same, if dhan folks think this could be a great addition.

Readers of this post: Do voice your opinion on the same.

**Comment by Iss-in** [comment]
I am also looking for the same, shifting from shoonya to dhan for killswitch feature. 
getting total brokerage for the day is really a required feature

**Comment by Iss-in** [comment]
@dhan-oss yes getting pnl is straightforward, but there is no way to get total brokerage for the day. Ideally "TotalBrokerage" for the day should be part of response for get_fund_limits api

**Comment by dhan-oss** [comment]
@Iss-in Will look into adding this. Fund Limit might not be the best possible API where this can be added, but rather somewhere charges can be posted on trade basis.

**Comment by Iss-in** [comment]
@dhan-oss any update on this ? 
sad to see such a basic and simple feature being ignored

**Comment by abhisheksoni27** [comment]
+1 here, this can be quite useful.

**Comment by dhan-oss** [comment]
Hey @Iss-in @abhisheksoni27 

You can use the Historical Trade Statement for the same - [here](https://dhanhq.co/docs/v2/statements/#trade-history). This is not added on the python library yet, but is quite simple to integrate.

**Comment by Iss-in** [comment]
@dhan-oss I checked this and this is historical trade statement only, There is no information about trades for the current day which is what this issue was created for. 
Please reopen the issue

**Comment by abhisheksoni27** [comment]
@dhan-oss have we given up on the suggestions of the actual users of this module? Are we only following the rule books announced in the headquarters?
